frontend/src/metabase-lib/queries/structured/Aggregation.ts

```diff
@@ -12,7 +12,10 @@ import Metric from "metabase-lib/metadata/Metric";
 import StructuredQuery from "../StructuredQuery";
 import Dimension, { AggregationDimension } from "../../Dimension";
 import MBQLClause from "./MBQLClause";
+
 const INTEGER_AGGREGATIONS = new Set(["count", "cum-count", "distinct"]);
+const ORIGINAL_FIELD_TYPE_AGGREGATIONS = new Set(["min", "max"]);
```

Comment:
This is from testing that only `min` and `max` will result in the value with the base field's `base_type` other aggregations will always result in a number.

<img width="364" alt="image" src="https://user-images.githubusercontent.com/1937582/221925571-9dae9b03-e198-46c4-b990-465baaafe26b.png">